tests/testthat/test-splatPop-simulate.R

```diff
@@ -2,13 +2,13 @@ context("splatPopSimulate")
 
 if (requireNamespace("VariantAnnotation", quietly = TRUE) &&
     requireNamespace("preprocessCore", quietly = TRUE)) {
-
+    library(VariantAnnotation)
```

Comment:
Do we need to load the library? `mockVCF()` should work with it being loaded right?